.github/workflows/ci.yaml

```diff
@@ -0,0 +1,36 @@
+name: Tests
+
+on:
+  - push
+  - pull_request
+
+env:
+  GOPATH: ${{ github.workspace }}/go
+
+jobs:
+  test-and-verify:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.14
```

Comment:
I think we're at 1.15 now (at least in CA).

Reply:
Travis is at 1.14

https://github.com/kubernetes/autoscaler/blob/master/.travis.yml#L9

Let's go with 1.14. Then we can update to 1.15.

Reply:
I think travis being on 1.14 is a bug (we must have forgotten - again - to bump it when bumping go version). https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/go.mod#L9

Reply:
Ok, whatever. Flipped to 1.15.